types/remote-data.d.ts

```diff
@@ -97,6 +97,7 @@ interface RemoteDataApiRequest {
 	block_name: string;
 	query_inputs: RemoteDataQueryInput[];
 	query_key: string;
+	post_id: ?( number | string );
```

Comment:
This is the published type in the docs, but the package types indicate `number | undefined`, which we modify to `number | null`.

```suggestion
	post_id: number | null;
	query_inputs: RemoteDataQueryInput[];
	query_key: string;
```

Reply:
hmm - confusing, must have done it based on the docs.

Reply:
ah, it fails: https://github.com/Automattic/remote-data-blocks/actions/runs/18285326911/job/52058739549

Reply:
My fault, in that case change to `number | string | null`